workflows/subclustering/🍒_RelevanceAI_Subclustering.ipynb

```diff
@@ -0,0 +1,16777 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from relevanceai.utils.datasets import get_ecommerce_dataset_encoded
</pre><p>A workflow is not meant to be creating a new dataset.  It is meant to be run from top to bottom. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/workflows/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>